src/app/modules/item/components/permissions-edit-dialog/permissions-edit-dialog.component.ts

```diff
@@ -20,13 +21,20 @@ export interface PermissionsEditData {
   templateUrl: './permissions-edit-dialog.component.html',
   styleUrls: [ './permissions-edit-dialog.component.scss' ]
 })
-export class PermissionsEditDialogComponent {
+export class PermissionsEditDialogComponent implements OnInit {
 
   @Input() data?: PermissionsEditData;
   @Output() close = new EventEmitter<Permissions>();
+  @Input() targetType: TypeFilter = 'Users';
 
   public visible = false;
 
+  targetTypeString = '';
+
+  ngOnInit(): void {
+    this.targetTypeString = this.targetType.slice(0, -1).toLowerCase();
```

Comment:
Wow that's is very hacky. If the filter is changed for any reason, the text here will break with nobody noticing it. It should be based on a solid state, a constant/enum or whatever which cannot be changed without breaking somewhere with the types.

Reply:
fixed